pallets/eth-bridge/src/migration.rs

```diff
@@ -40,7 +40,7 @@ impl<T: Config<I>, I: 'static> OnRuntimeUpgrade for EthBridgeMigrations<T, I> {
         );
         let mut consumed_weight = Weight::zero();
 
-        if onchain < 2 {
+        if EthBlockRangeSize::<T, I>::get() == 0 {
```

Comment:
why are we deviating from using the built-in method (pallet version)?

Reply:
cause on new instances that are included in latest version, the migration is not run and the value is not set

Reply:
When new instances are added, we would need to do a forkless upgrade which would change the pallet version so we could use the migration convention i think.

Reply:
The migration will not be triggered as the new instance has the latest version of the pallet. I leave this as is for not, lets sync if we need to review this.